adr/0002-workflow-integration-points.md

```diff
@@ -0,0 +1,36 @@
+# 2. Provide workflow integration points
+
+Date: 2018-04-27
+
+## Status
+
+Proposed
+
+## Context
+
+A [workflow system](https://en.wikipedia.org/wiki/Workflow_management_system) to handle the creation, 
+processing and orchestration of customized workflows is a required component of Libero's currently proposed design.
+
+Users will each have their individual needs and requirements and therefore may decide to use their 
+workflow solution of choice. 
+
+It is therefore important to define clear integration points between a [workflow system](https://en.wikipedia.org/wiki/Workflow_management_system) 
+and the rest of the Libero architecture.
+
+## Decision
+
+The workflow solution should provide:
+
+- HTTP endpoint(s) to allow the triggering of workflows
+- A queue to allow the triggering of workflows
```

Comment:
probably saw it earlier, but any internal queue should likely be hidden is the same job is performed by the HTTP API. Being asynchronous (a workflow is just started rather than executed within the `POST` lifecycle) there should be no particular need to expose the queue to clients.